### PR TITLE
TM-1404 projected vacancies export

### PR DESCRIPTION
**Pull request by BaldwinJordan**

### CODE REVIEW

**Comment by BaldwinJordan** [comment]
It could probably be refactored better for the available positions one as well but I couldn't think of how so for now I got it working

File: talentmap_api/fsbid/services/projected_vacancies.py [hunk 1]
```diff
@@ -36,14 +41,67 @@ def get_projected_vacancies(query, jwt_token, host=None):
...
+    writer.writerow([
+        smart_str(u"Position"),
+        smart_str(u"Position Number"),
+        smart_str(u"Skill"),
+        smart_str(u"Grade"),
+        smart_str(u"Bureau"),
+        smart_str(u"Post City"),
+        smart_str(u"Post Country"),
+        smart_str(u"Tour of Duty"),
+        smart_str(u"Languages"),
```
Comment (mjoyce91) [inline talentmap_api/fsbid/services/projected_vacancies.py]: Take a look at languages

Screen Shot 2019-11-22 at 9 12 27 AM

Comment (mjoyce91) [reply talentmap_api/fsbid/services/projected_vacancies.py]: Looks like this needs to be updated for the available_position export too - we just never had any languages in our dataset. This should be a comma separated list of the `language` prop from each object. Example:

`[ { ..., language: 'Spanish', ...}, { ..., language: 'French', ... } ]` becomes: Spanish, French

Comment (BaldwinJordan) [reply talentmap_api/fsbid/services/projected_vacancies.py]: Sorry this took so long, should parse better now

**Pull request merged**